Cargo.toml

```diff
@@ -7,7 +7,7 @@ authors = ["Andy Grover <agrover@redhat.com>", "Mulhern <amulhern@redhat.com>",
 dbus = "0.5.2"
 clap = "1"
 nix = "0"
-devicemapper = "0.5"
+devicemapper = { path = "/home/mulhern/my-contributions/devicemapper-rs" }
```

Comment:
It won't pass this way. But, I guess I could see if it would pass by changing this to a git reference.